app/components/services/servicesSubPages/ComprehensiveOrthopedicCare/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import ServicesContentBlock from "../../servicesContentBlock";
+import { COMPREHENSIVE_ORTHOPEDIC_CARE } from "@/Constants/servicesData";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+
+const ComprehensiveOrthopedicCare = () => {
+  const router = useRouter();
+  return (
+    <>
+      <ServicesContentBlock
```

Comment:
there is no gap between the cards 
![image](https://github.com/user-attachments/assets/1b26c9eb-28bb-4797-8041-4ed3854811a9)